internal/handlers/common/msg_listcommands.go

```diff
@@ -95,6 +95,10 @@ var Commands = map[string]command{
 		Help:    "Drops production database.",
 		Handler: (handlers.Interface).MsgDropDatabase,
 	},
+	"explain": {
+		Help:    "Explain diagnostic command.",
```

Comment:
Note that other command's help text actually describes the behavior

Reply:
> Note that other command's help text actually describes the behavior

changed to `Display the execution plan.`